Orange/widgets/gui.py

```diff
@@ -34,6 +34,7 @@
     CalendarWidgetWithTime, DateTimeEditWCalendarTime,
     ControlGetter, VerticalScrollArea, ProgressBar,
     ControlledCallback, ControlledCallFront, ValueCallback, connectControl,
+    is_macstyle
```

Comment:
Isn't `Orange/widget/gui.py` only for backward compatibility and Orange-specific additions? If so, the new `is_macstyle` from `widget-base` should not be added here.